src/main/java/data/workouts/WorkoutList.java

```diff
@@ -41,4 +44,34 @@ public Workout createAndAddWorkout(String userArgument) throws ArrayIndexOutOfBo
     public boolean checkIfRepsValueIsValid(int userRepsInput) {
         return userRepsInput >= 1;
     }
+
+    public void listWorkout() {
+        int index = 0;
+        int totalPrints = workoutsList.size();
+
+        while (totalPrints > MAX_DISPLAY) {
+            index = continuousPrinting(index, 10);
+            totalPrints -= 10;
+            System.out.println("Do you want to view more workouts? [yes/no]");
+            Scanner in = new Scanner(System.in);
+            String input = in.nextLine();
+            if (!input.equals("yes")) {
+                return;
+            }
+        }
+        if (totalPrints > 0) {
```

Comment:
Perhaps there can be an empty line before Line 62 to separate the loop and the conditional statement.